sdk/eventhub/tests.yml

```diff
@@ -12,9 +12,6 @@ extends:
       EnvVars:
         AZURE_TEST_RUN_LIVE: 'true'
         AZURE_SUBSCRIPTION_ID: $(EVENTHUB_SUBSCRIPTION_ID)
-        # Set fake authority host to ensure Managed Identity fail for Default Azure Credential
-        # so "execute samples" step correctly picks up Powershell credential.
-        AZURE_POD_IDENTITY_AUTHORITY_HOST: "FakeAuthorityHost"
```

Comment:
you can undo this here and rebase these changes https://github.com/Azure/azure-sdk-for-python/pull/38596#event-15353646101 :)